jsonnetfile.json

```diff
@@ -9,6 +9,16 @@
                 }
             },
             "version": "master"
+        },
+        {
+            "name": "grafonnet",
```

Comment:
why do we have 2 dependencies with same name?
Do we need both? If so, please rename.

Reply:
ack, will rename.